testdata/p4_16_samples_outputs/table-entries-ternary-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,100 @@
+updates {
```

Comment:
Repeat comments on this file of things that were present in earlier ones, too: (value & mask) != mask in one entry.  Should field be present or absent in file if it is completely don't-care.